docs/sources/setup-grafana/configure-security/configure-scim-provisioning/_index.md

```diff
@@ -0,0 +1,127 @@
+
+{{< admonition type="note" >}}
+Available in [Grafana Enterprise](../../../introduction/grafana-enterprise/) and [Grafana Cloud Advanced](/docs/grafana-cloud/).
+{{< /admonition >}}
+
+{{< admonition type="note" >}}
+This feature is behind the `enableSCIM` feature toggle.
+You can enable feature toggles through configuration file or environment variables.
+
+For more information, refer to the [feature toggles documentation](/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles).
+{{< /admonition >}}
+
+## Benefits
+
+{{< admonition type="note" >}}
+SCIM provisioning only works SAML authentication.
+Other authentication methods aren't supported.
+{{< /admonition >}}
+
+SCIM offers several advantages for managing users and teams in Grafana:
+
+- **Automated user provisioning**: Automatically create, update, and disable users in Grafana when changes occur in your identity provider
+- **Automated team provisioning**: Automatically create, update, and delete teams in Grafana based on groups in your identity provider
+- **Reduced administrative overhead**: Eliminate manual user management tasks and reduce the risk of human error
+- **Enhanced security**: Automatically disable access when users leave your organization
+
+## Supported identity providers
+
+The following identity providers are supported:
+
+- [Azure AD](../configure-authentication/azuread/)
+- [Okta](../configure-authentication/saml/)
+
+## How it works
+
+The synchronization process works as follows:
+
+1. Configure SCIM in both your identity provider and Grafana
+2. Your identity provider sends SCIM requests to the Grafana SCIM API endpoint
+3. Grafana processes these requests to create, update, or deactivate users and teams, and synchronize team memberships
+
+## Comparison with other sync methods
+
+Grafana offers several methods for synchronizing users, teams, and roles.
+The following table compares SCIM with other synchronization methods to help you understand its advantages:
+
+| Sync Method                                                                    | Users | Teams | Roles | Automation | Key Benefits                                                             | Limitations                                                  | On-Prem | Cloud |
+| ------------------------------------------------------------------------------ | ----- | ----- | ----- | ---------- | ------------------------------------------------------------------------ | ------------------------------------------------------------ | ------- | ----- |
+| SCIM                                                                           | ✅    | ✅    | ⚠️    | Partial    | Complete user and team lifecycle management with automatic team creation | Requires SAML authentication; uses Role Sync for basic roles | ✅      | ✅    |
```

Comment:
It says automation partial - but in the summary below, it says full automation

Reply:
Good catch, claiming that "SCIM is the only method that fully automates user and team provisioning" is still technically correct, as only role sync is not included, I will think how can we make it clear and less conflicting.

Reply:
I think you should say Full here - since for the ticked columns (user and teams), the automation is full - right?